hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/ReplicatedMapService.java

```diff
@@ -75,11 +75,13 @@
 public class ReplicatedMapService implements ManagedService, RemoteService, EventPublishingService<Object, Object>,
         MigrationAwareService, SplitBrainHandlerService, StatisticsAwareService {
 
+
     /**
      * Public constant for the internal service name of the ReplicatedMapService
      */
     public static final String SERVICE_NAME = "hz:impl:replicatedMapService";
 
+    public static final int INVOCATION_TRY_COUNT = 3;
```

Comment:
Since anti-entropy is a periodic task, instead of trying it 250 (default) times we are hoping to retry this invocation in the next execution. I believe retrying 3 times will cover the intermittent connection problems, even if not, as I said it will retried on the next execution.